src/software/simulation/simulator.h

```diff
@@ -64,8 +64,13 @@ class Simulator
      * will have the given field, with no robots or ball.
      *
      * @param field The field to initialize the simulation with
+     * @param physics_time_step The time step used to simulated physics
+     * and robot primitives. The default value is approximately how often
+     * the robot firmware updates.
      */
-    explicit Simulator(const Field& field);
+    explicit Simulator(const Field& field,
+                       const Duration& physics_time_step = Duration::fromSeconds(1.0 /
+                                                                                 200.0));
```

Comment:
Can we make the default value a static const class member so that it has some meaning? maybe `static constexpr double STABLE_PHYSICS_TIME_STEP_SEC = 1.0 / 200.0`?

Reply:
done, made `PHYSICS_TIME_STEP_SECONDS` with a comment justifying the value.